tools/c7n_azure/c7n_azure/session.py

```diff
@@ -132,7 +132,8 @@ def resource_api_version(self, resource_id):
         provider = resource_client.providers.get(namespace)
 
         rt = next((t for t in provider.resource_types
-            if t.resource_type == str(resource_type).split('/')[-1]), None)
+           if provider_resource_type_comparator(t.resource_type.lower(), resource_type)), None)
```

Comment:
I'd update the comparator to be insensitive and remove the `lower` here so all of that is encapsulated in one place.  Someone else that uses this method might not know they have to lowercase the input.

Reply:
Will make fix above